### PR TITLE
remove EJB interop tests.

### PR DESCRIPTION
**Pull request by gurunrao**
Signed-off-by: gurunandan <gurunandan.rao@oracle.com>

### CODE REVIEW

File: install/jakartaee/bin/ts.jte [hunk 1]
```diff
@@ -1287,7 +1287,7 @@ command.testExecuteAppClient2= \
         PATH="${javaee.home.ri}/nativelib" \
         APPCPATH=${ts.home}/lib/tsharness.jar${pathsep}${ts.home}/lib/cts.jar${pathsep}${ts.home}/lib/riinterceptors.jar \
 	TZ=${tz} \
-        ${RI_JAVA_HOME}/bin/java \
```
Comment (alwin-joseph) [inline install/jakartaee/bin/ts.jte:1290]: Just to confirm, since interop tests is now removed we wont need the RI_JAVA_HOME correct ?

Comment (scottmarlow) [reply install/jakartaee/bin/ts.jte:1290]: I think that jws + jaxws test folders also use forward/reverse testing with the RI.

Comment (gurunrao) [reply install/jakartaee/bin/ts.jte:1290]: RI_JAVA_HOME was  used for EJB interop tests, jdk installation separation.

Comment (alwin-joseph) [reply install/jakartaee/bin/ts.jte:1290]: @gurunrao Can you suggest if we can retain the RI_JAVA_HOME, default it to JAVA_HOME so the jaxws tests can make use of it, if required.

Comment (gurunrao) [reply install/jakartaee/bin/ts.jte]: have add back RI_JAVA_HOME, so that in future jaxws tests can make used of them.

**Comment by scottmarlow** [comment]
@gurunrao do the interop references in release/tools/build.xml need to be removed? I assume that would include jakartaee-interop + jakartaee-interop-security.

**Comment by gurunrao** [comment]
> @gurunrao do the interop references in release/tools/build.xml need to be removed? I assume that would include jakartaee-interop + jakartaee-interop-security.

@scott have removed jakartaee-interop + jakartaee-interop-security from release/tools/build.xml

**Comment by scottmarlow** [comment]
Thanks Guru!

**Pull request merged**